### PR TITLE
Add joshjon-nocturnal v1.1.1

### PR DESCRIPTION
**Pull request by joshjon**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ x There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/nocturnal-screen.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'nocturnal-screen' do
+  version '1.1'
+  sha256 'b3ab92067e1fa999ef09cadf6a26e23c1a46e6595fd64d9a1b614225fdbd4681'
+
+  url 'https://github.com/joshjon/nocturnal/releases/download/v1.1/Nocturnal-1.1.zip'
```
Comment (miccal) [inline Casks/nocturnal-screen.rb:5]: Please change this to:
```
url "https://github.com/joshjon/nocturnal/releases/download/v#{version}/Nocturnal-#{version}.zip"
```

Comment (joshjon) [reply Casks/nocturnal-screen.rb]: Done

File: Casks/nocturnal-screen.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'nocturnal-screen' do
```
Comment (vitorgalvao) [inline Casks/nocturnal-screen.rb:1]: This does not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). Needs to be `joshjon-nocturnal`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by ran-dall — APPROVED** [review]

File: Casks/nocturnal-screen.rb (hunk 2 above)
Comment (joshjon) [reply Casks/nocturnal-screen.rb]: The reason for this is mainly becuase I dont have an official vendor name and preferred to not include my personal name in the cask title. However, if this is cumpolsary then all good.

**Pull request merged**